README.md

```diff
@@ -44,7 +44,7 @@ You can customize the error message displayed in two ways:
 ```javascript
 import { isUrl } from '@lemoncode/fonk-is-url-validator';
 
-isUrl.setErrorMessage('El campo debe de ser numérico');
+isUrl.setErrorMessage('This field should be a URL');
```

Comment:
Sorry realized is correct in Spanish (up above says porting to Spanish, just showing how to set multilanguage), but totally agree the message is copy pasted XDD, thanks

can you update it to

'El campo no es una URL válida'

Muchas gracias ;)